rotkehlchen/api/v1/schemas.py

```diff
@@ -3694,3 +3694,27 @@ def make_calendar_query(
             identifiers=data['identifiers'],
         )
         return {'filter_query': filter_query}
+
+
+class CalendarReminderCommonEntrySchema(Schema):
+    event_id = fields.Integer(required=True)
+    secs_before = fields.Integer(
+        required=True,
+        validate=webargs.validate.Range(min=1, error='secs_before has to be bigger than 0'),
+    )
+
+
+class NewCalendarReminderSchema(CalendarReminderCommonEntrySchema):
+    @post_load
+    def make_calendar_entry(self, data: dict[str, Any], **_kwargs: dict[str, Any]) -> dict[str, Any]:  # noqa: E501
+        return {
+            'reminder': ReminderEntry(
+                identifier=data.get('identifier', 0),
```

Comment:
why give a defautl value at all here? When can it not be given? And why should the default value be 0?